packages/skill-api/src/core/services/process-stripe-webhook.ts

```diff
@@ -26,7 +29,19 @@ const webhookSecret = process.env.STRIPE_WEBHOOK_SECRET
 
 const METADATA_MISSING_SITE_NAME = 'metadata-missing-site-name'
 
-type PaymentOptions = {stripeCtx: {stripe: Stripe}}
+// type PaymentOptions = {stripeCtx: {stripe: Stripe}}
+
+const getStripeClient = (paymentOptions: PaymentOptions | undefined) => {
+  return paymentOptions?.providers.stripe?.paymentClient
+}
+
+const constructFallbackStripePaymentOptions = (
+  stripe: Stripe,
+): PaymentOptions => {
+  return defaultPaymentOptions({
+    stripeProvider: StripeProvider({defaultStripeClient: stripe}),
+  })
+}
```

Comment:
this module processes stripe webhooks, so it can reach directly for the stripe client.

since most of the apps are still using the stripe module instead of configuring payment providers, this other helper function makes it easy to fallback on that client while conforming to the new provider-based API